snippets/csharp/language-reference/operators/DivisionExamples.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+
+namespace operators
+{
+    public static class DivisionExamples
+    {
+        public static void Examples()
+        {
+            IntegerDivision();
+            IntegerAsFloatingPointDivision();
+            FloatingPointDivision();
+            DivisionAssignment();
+        }
+
+        private static void IntegerDivision()
+        {
+            // <SnippetInteger>
+            Console.WriteLine(13 / 5);    // output: 2
+            Console.WriteLine(-13 / 5);   // output: -2
+            Console.WriteLine(13 / -5);   // output: -2
+            Console.WriteLine(-13 / -5);  // output: 2
+            // </SnippetInteger>
+        }
+
+        private static void IntegerAsFloatingPointDivision()
+        {
+            // <SnippetIntegerAsFloatingPoint>
+            Console.WriteLine(13 / 5.0);       // output: 2.6
+
+            int a = 13;
+            int b = 5;
+            Console.WriteLine(a / (double)b);  // output: 2.6
```

Comment:
I'd consider casting `a` instead of `b` to show that if *either* the numerator or the denominator is a double, the result is a double. (The first sample has an integer numerator divided by a floating point denominator).